.github/workflows/release.yaml

```diff
@@ -56,10 +56,140 @@ jobs:
         destination: tanzu-cli-admin-plugins
         credentials: ${{ secrets.GCP_BUCKET_SA }}
 
+    - name: Generate details about repo
+      id: generate-repo-details
+      shell: bash
+      run: |
+        echo "::set-output name=repo-name::$(echo '${{ github.repository }}' | awk -F '/' '{print $2}')"
+        echo "::set-output name=repo-org::$(echo '${{ github.repository }}' | awk -F '/' '{print $1}')"
+
+    - name: Get last release info
+      uses: octokit/request-action@v2.x
+      id: get-last-release
+      with:
+        route: GET /repos/{owner}/{repo}/releases/latest
+        owner: ${{ steps.generate-repo-details.outputs.repo-org }}
+        repo: ${{ steps.generate-repo-details.outputs.repo-name }}
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Generate details for creating release notes
+      id: generate-release-details
+      shell: bash
+      run: |
+        LAST_TAG=${{ fromJson(steps.get-last-release.outputs.data).tag_name }}
+        echo "last tag - $LAST_TAG"
+        LAST_TAG_SHA=$(git rev-list -n 1 ${{ fromJson(steps.get-last-release.outputs.data).tag_name }})
+        echo "Last tag SHA - $LAST_TAG_SHA"
+        END_SHA=$(git rev-parse HEAD)
```

Comment:
Do we need to iterate over sha array or can we get END_SHA from $(git rev-parse -n 1 RELEASE_VERSION) because the release tag has already been pushed before this job gets triggered?

Reply:
Originally was planning to take the commit after last release tag commit. Hence loop to find the correct sha. Based on discussion, making it simpler by using last release tag sha as the start sha.